### PR TITLE
Release/0.4.0

### PR DESCRIPTION
**Pull request by mblomdahl**
Creating a release, to establish a baseline for login.libris.kb.se. Let's make sure to have a 0.4.1 ready by lunch tomorrow so we can get the final features in place for early adopters!

Changelog for 0.4.0:

* Added `flask import_data` CLI tool for pulling data from legacy systems (#38, #43)
* Styling and usability improvements (#6, #22)
* Applied restrictions on anonymous users and non-admins (#48)
* Added new type of permission, "being the cataloging admin for a collection" (#40)
* Support for dev deployment on login.libris.kb.se (#39)

### CODE REVIEW

File: xl_auth/templates/public/home.html [hunk 1]
```diff
@@ -1,39 +1,68 @@
...
-                <p>Sleek, intuitive, and powerful mobile-first front-end framework for faster and
-                    easier web development.</p>
-                <p><a class="btn btn-default" href="http://getbootstrap.com/">Official website
-                    &raquo;</a></p>
+            {% if current_user and current_user.is_authenticated %}
+                <div class="jumbotron">
+
+                    <h1>{{ _('Welcome to xl_auth') }}</h1>
+
+                    <p>This is a starter Flask template. It includes Bootstrap 3, jQuery 2, Flask-SQLAlchemy,
```
Comment (mxtthias) [inline xl_auth/templates/public/home.html]: Should this still be here?

Comment (oBlissing) [reply xl_auth/templates/public/home.html]: #60

File: ansible/roles/common/tasks/configure-system.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+
+- name: set timezone to europe/stockholm
+  copy: src=/usr/share/zoneinfo/Europe/Stockholm dest=/etc/localtime
+
+- name: add 'root' user to 'wheel'
```
[the remark this answers is not in the document]
Comment (mblomdahl) [reply ansible/roles/common/tasks/configure-system.yml]: @mxtthias  You're absolutely right! Not needed. Some optimization stuff imported from my jenkins-up repo, methinks. Only used with SSH pipelining (not enabled in ansible.cfg). But it's useful when you have lots of tasks on remote servers with slow/high-latency connections. Nothing for us. 👍

**Comment by mblomdahl** [comment]
@mxtthias About the 
> this branch is currently missing the stuff that actually solves #48 (which is pending approval). :)

I think it would be of great benefit to roll out a baseline version ASAP, so we know what works (and doesn't) it terms of CD procedures. If there's nothing wrong with current 0.4.0, how about merging this, rolling it out from your computer, and spend tomorrow morning with Ola fixing issues with #56, you doing the #59 clean-ups, and me adding the #41 functionality? Then we can roll out 0.4.1 in the afternoon, confident that we already have a working deployment/upgrade procedure in place.

**Pull request merged**